test/GCClassic/parallel/parallelTestExecute.sh

```diff
@@ -184,6 +184,10 @@ for runDir in *; do
             # Remove any leftover files in the run dir
             ./cleanRunDir.sh --no-interactive >> "${log}" 2>&1
 
+	    # Change time cycle flag in HEMCO_Config.rc from EFYO to CYS,
+	    # to allow missing specis to be set a default value.
```

Comment:
Typo in species

Reply:
@lizziel.  Thanks. I'll fix that in a subsequent commit.

Reply:
Now fixed in commit 549674d25.